boards/arm/nucleo_f030r8/nucleo_f030r8_defconfig

```diff
@@ -28,7 +22,6 @@ CONFIG_BOOTLOADER_SRAM_SIZE=8
 CONFIG_SERIAL=y
 CONFIG_SERIAL_HAS_DRIVER=y
 CONFIG_UART_INTERRUPT_DRIVEN=y
-CONFIG_UART_STM32=y
 # enable USART2 - passthrough to STLINK v2 connector
 CONFIG_UART_STM32_PORT_2=y
```

Comment:
Since this is a board default configuration, can UART_2 activation be moved to Kconfig.defconfig under condition `if UART_CONSOLE`

Reply:
Done.